docs/sources/release-notes/v2-0.md

```diff
@@ -0,0 +1,203 @@
+
+TraceQL requires Tempo’s Parquet columnar format, which is the default block format in Tempo 2.0. For more information, refer to the Tempo [Apache Parquet block format](https://grafana.com/docs/tempo/latest/configuration/parquet/) documentation.
+
+### TraceQL query editor in Grafana
+
+To make it easier to run and visualize the results of TraceQL queries, we’ve added a TraceQL query editor to Grafana’s Tempo data source. The editor is available in Grafana’s Explore interface.
+
+<p align="center"><img src="../traceql/assets/query-editor-results-span.png" alt="Query editor showing span results" /></p>
+
+The query editor helps you learn TraceQL by offering autocomplete.
+
+<p align="center"><img src="../traceql/assets/query-editor-auto-complete.png" alt="Query editor showing the auto-complete feature" /></p>
+
+For more information, refer to the [TraceQL query editor documentation](https://grafana.com/docs/tempo/latest/traceql/query-editor.md).
+
+### Apache Parquet block format is the new default
+
+The experimental Apache Parquet columnar block format introduced in Tempo v1.5 has been promoted to stable and is now the default trace storage for Tempo 2.0. This format enables increased speed and efficiency for Tempo search.
+In our own environments, we’re seeing that it allows us to scan trace data at rates of 1 TB/s (with some bursts as high as 4 TB/s), a massive improvement over the **XXX** rates that were more typical with our previous storage format. Switching to the Parquet block format also allows users to leverage the large ecosystem of existing Parquet tools and libraries to process and transform their trace data.
+
+The Parquet block format can be used as a drop-in replacement for Tempo’s existing block format. No data conversion or upgrade process is necessary. As soon as the Parquet format is enabled, Tempo starts writing data in that format, leaving existing data as-is.
+
+Tempo search will return results from both the legacy and Parquet block formats. However, TraceQL will only search Parquet. Search on legacy blocks is deprecated and will be removed in Tempo 2.1.
+For more information, refer to the Parquet documentation.
+
+Refer to the [Parquet documentation]({{< relref "../configuration/parquet/" >}}).
+
+### Search and metrics-generator enabled by default
+
+Search and [metrics-generator]({{< relref "../metrics-generator/" >}}) are now enabled by default in Tempo 2.0, and their corresponding configuration parameters (search_enabled and metrics_generator_enabled ) have been removed).
+
+We’ve been excited by the feedback we’ve gotten on both features and are satisfied with their stability so we want all newcomers to Tempo to have access to them from day 1.
+
+>**NOTE:** The metrics-generator is not turned on by default for Grafana Cloud customers due to the potential cost implications.
```

Comment:
Will Cloud customers still have to contact support to enable metrics-generator? Or do they just enable in a configuration file some where?